library/core/src/fmt/rt.rs

```diff
@@ -151,7 +155,10 @@ impl Argument<'_> {
     }
     #[inline]
     pub const fn from_usize(x: &usize) -> Argument<'_> {
-        Argument { ty: ArgumentType::Count(*x) }
+        if *x > u16::MAX as usize {
+            panic!("Formatting argument out of range");
```

Comment:
How about `#[track_caller]` so it won't report an internal location?

Reply:
Done!